rmgpy/rmg/main.py

```diff
@@ -826,11 +824,13 @@ def makeSeedMech(self,firstTime=False):
                     q += 1
                 self.name = name + str(q)
         
-        if firstTime and not os.path.exists(os.path.join(currentDir,'seed')): #if seed directory does not exist make it
-            os.system('mkdir seed')
+        seedDir = os.path.join(self.outputDirectory,'seed')
+        
+        if firstTime and not os.path.exists(seedDir): #if seed directory does not exist make it
+            os.system('mkdir '+seedDir)
```

Comment:
Good point, Python documentation recommends `shutil.rmtree()` for non-empty directories. The bigger issue is that I don't think `rm -rf` works on Windows.